manage_proj_create_page.php

```diff
@@ -49,15 +49,16 @@
 
 auth_reauthenticate();
 
-access_ensure_global_level( config_get( 'create_project_threshold' ) );
+$f_parent_id = gpc_get( 'parent_id', null );
+$t_threshold = is_null($f_parent_id) ? 'create_project_threshold' : 'create_subproject_threshold';
```

Comment:
Replace `is_null($f_parent_id)` with `is_null( $f_parent_id )` - note the spaces.